extensions/1bookmark/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 120,
+  "singleQuote": false
```

Comment:
@pernielsentikaer Is it mandatory to use this for prettier settings?

Reply:
That is what we use for all extensions, yes. We would like to keep that consistent 🙂

Reply:
I changed it as default.